src/express.js

```diff
@@ -7,17 +7,23 @@ const compression = require('compression');
 const { healthcheckRoute } = require('./healthcheck.js');
 const { handleNoMatch, handleError } = require('./errors.js');
 
-function start(app, config) {
+function notifySuccess(app, callback) {
+  callback && callback(null, { app });
+}
+
+function start(app, config, callback) {
     if (String(process.env.ENABLE_CLUSTER).toLowerCase() === 'true' && cluster.isMaster) {
         os.cpus().forEach(c => cluster.fork());
         cluster.on('exit', function fork(worker) {
             log.fatal(`Worker ${worker.id} died. Starting a new one.`);
             cluster.fork();
         });
+        callback();
```

Comment:
I'm not very familiar with express or clusters, so please double check that these callback calls are in the right spot here.

Reply:
This should be the right spot... we will know if it breaks.